src/Miningcore/DataStore/Postgres/Scripts/db_change_20211004.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS payment_schedule;
+
+CREATE TABLE poolstate
+(
+	poolid TEXT NOT NULL PRIMARY KEY,
+	hashvalue DOUBLE PRECISION NOT NULL DEFAULT 0,
```

Comment:
can we use the decimal type instead of double precision?  this could be a very tiny fraction.
https://www.postgresql.org/docs/11/datatype-numeric.html